config/jobs/kubernetes/minikube/minikube.yaml

```diff
@@ -0,0 +1,28 @@
+presets:
+- env:
+  - name: GCS_CREDS
+    value: /root/.config/k8s-minikube-20e2437ce844.json
+  volumes:
+  - name: k8s-minikube-build-gcs
+    secret:
+      secretName: k8s-minikube-build-gcs
+      items:
+      - key: k8s-minikube-20e2437ce844.json
+        path: k8s-minikube-20e2437ce844.json
+  volumeMounts:
+  - name: k8s-minikube-build-gcs
+    mountPath: /root/.config/gcloud/k8s-minikube-20e2437ce844.json
+    readOnly: true
+
+presubmits:
+  kubernetes/minikube:
+  - name: pull-linux-kvm
+    always_run: false
```

Comment:
Consider also adding the field `optional: true` so that when the job is manually run (with `/test pull-linux-kvm`) it will post a status to GitHub, but Tide will ignore the status when determining if the PR is mergeable.